internal/pkg/deploy/cloudformation/app.go

```diff
@@ -220,11 +220,17 @@ func (cf CloudFormation) getResourcesForStackInstances(app *config.Application,
 	return regionalResources, nil
 }
 
+// AddWorkloadToAppOpt allows passing optional parameters to AddServiceToApp.
+type AddWorkloadToAppOpt func(*stack.AppResourcesService)
+
+// AddWorkloadToAppOptWithoutECR adds a workload to app without creating an ECR repo.
+func AddWorkloadToAppOptWithoutECR(config *stack.AppResourcesService) { config.WithECR = false }
```

Comment:
nit: most functional options i've seen return an option function:
```suggestion
func AddWorkloadToAppWithoutECR(config *stack.AppResourcesService) AddWorkloadToAppOpt {
  return func(config *stack.AppResourcesService) {
    config.WithECR = false
  } 
}
```
i think it's mostly so that a user of the option could provide parameters if they were necessary. in this case, they're not, so it doesn't really matter either way!